content/en/docs/tasks/tools/install-kubectl-linux.md

```diff
@@ -314,6 +314,57 @@ Below are the procedures to set up autocompletion for Bash, Fish, and Zsh.
    rm kubectl-convert kubectl-convert.sha256
    ```
 
+### Uninstall kubectl using package management
+{{< tabs name="other_kubectl_uninstall" >}}
+{{% tab name="apt-get" %}}
+If you are on Ubuntu or another Debian based Linux distribution
+
+```shell
+sudo apt-get purge kubectl
+sudo apt-get autoremove
+```
+
+The executable binaries are stored in the ```/usr/bin or usr/local/bin```, library files in ```usr/lib``` and configuration files in ```/etc``` 
```

Comment:
For binary files and other path, you can mention here command - 

```bash
which kubectl
```
or 

```bash
whereis kubectl

Reply:
@utkarsh-singh1 

I have changed everything. Could you please have a look?

Thank you

Reply:
@utkarsh-singh1 

I tried the uninstallation of the kubectl using the command using 
```shell
sudo rm $(which kubectl)
```

It was uninstalled successfully. But, when I tried to install the kubectl and tried to run it, the command was not recognised by the shell. I had to remove the kubectl completely using my package manager and install it again for it to run.